src/reducers/entities/channels.test.js

```diff
@@ -482,4 +494,94 @@ describe('channels', () => {
             });
         });
     });
+
+    describe('RECEIVED_CHANNEL_MODERATIONS', () => {
+        test('Should add new channel moderations', () => {
+            const state = deepFreeze({
+                channelsInTeam: {},
+                currentChannelId: '',
+                groupsAssociatedToChannel: {},
+                myMembers: {},
+                stats: {},
+                totalCount: 0,
+                membersInChannel: {},
+                channels: {
+                    channel1: {
+                        id: 'channel1',
+                        team_id: 'team',
+                    },
+                },
+                channelModerations: {},
+            });
+
+            const nextState = channelsReducer(state, {
+                type: ChannelTypes.RECEIVED_CHANNEL_MODERATIONS,
+                sync: true,
+                currentChannelId: 'channel1',
+                teamId: 'team',
+                data: {
+                    channelId: 'channel1',
+                    moderations: [
+                        {
+                            "name": "create_posts",
```

Comment:
This looks incorrect. I think the style checker might not be running correctly? It'd usually want this to be
```suggestion
                            name: 'create_posts',
```

Reply:
Also, should `'create_posts'` be a constant?

Reply:
PR to fix ESLint: https://github.com/mattermost/mattermost-redux/pull/1050